examples/pytorch/graphsage/graphsage_sampling.py

```diff
@@ -194,7 +208,16 @@ def run(proc_id, n_gpus, args, devices, data):
     train_nid = th.split(train_nid, len(train_nid) // n_gpus)[dev_id]
 
     # Create sampler
-    sampler = NeighborSampler(g, [args.fan_out] * args.num_layers)
+    sampler = NeighborSampler(g, [int(fanout) for fanout in args.fan_out.split(',')])
+
+    # Create PyTorch DataLoader for constructing blocks
+    dataloader = DataLoader(
+        dataset=train_nid.numpy(),
+        batch_size=args.batch_size,
+        collate_fn=sampler.sample_blocks,
+        shuffle=True,
+        drop_last=False,
+        num_workers=args.num_workers_per_gpu)
```

Comment:
can we have builtin sampler for neighbor sampling?

Reply:
What do you mean by builtin sampler?  Everything in C?

Reply:
Quan and I think we could stay with the current implementation first and gradually putting some common samplers to DGL package once we have more examples built up. @zheng-da what do you think?